3-cloud-computing/3.0.1-Cloud Service Ecosystems.md

```diff
@@ -0,0 +1,87 @@
+# 3.0.1 Cloud Service Ecosystems
```

Comment:
This is a great addition to the bootcamp. I'd recommend just moving the information from 3.0.1 over into "3.0-overview.md".

Reply:
It fit better as it's own subsection